libraries/serviceescalation.rb

```diff
@@ -38,12 +39,12 @@ class Serviceescalation < Nagios::Base
                   :escalation_period
 
     def initialize(name)
-      @service_description = name
-      @name                = name
+      @service_description, @name = name
       @contacts            = {}
       @contact_groups      = {}
       @host_name           = {}
       @hostgroup_name      = {}
+      @servicegroup_name   = {}
```

Comment:
within the definition format https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/objectdefinitions.html#serviceescalation 
I don't see the servicegroup_name directive....